rasa/core/brokers/pika.py

```diff
@@ -159,15 +159,35 @@ async def connect(self) -> None:
 
         self._exchange = await self._set_up_exchange(channel)
 
-    async def _connect(self) -> aio_pika.abc.AbstractRobustConnection:
+    def configure_url(self) -> Optional[Text]:
```

Comment:
Rename to `_configure_url` because we do not need to make this a part of PikaBrokers API.